Mathlib/Data/Finsupp/Notation.lean

```diff
@@ -91,7 +91,8 @@ unsafe instance instRepr {α β} [Repr α] [Repr β] [Zero β] : Repr (α →₀
       if p ≥ leadPrec then Format.paren ret else ret
 #align finsupp.has_repr Finsupp.instRepr
 
--- lean4#3497 causes a PANIC if we put this in `Mathlib.Data.DFinsupp.Notation` where it belongs
+-- This cannot be put in `Mathlib.Data.DFinsupp.Notation` where it belongs, since doc-strings
+-- can only be added/modified in the file where the declaration is defined.
```

Comment:
perhaps: "where the corresponding declaration is defined"?

Reply:
Thanks, I added "corresponding"!